# {"text": txt}
string input_json
---
string output_json
# {"status": "success", "file_path": "xxx/tts.wav"}
# {"status": "error", "message": "xxx"}
